nixos/modules/services/system/dbus.nix

```diff
@@ -122,6 +141,30 @@ in
 
     systemd.services.dbus.restartTriggers = [ configDir ];
 
+    systemd.user = mkIf config.systemd.user.dbus.enable {
+      services.dbus = {
+        description = "D-Bus User Message Bus";
+        requires = [ "dbus.socket" ];
```

Comment:
Mystery solved, @lethalman.

All units with ```DefaultDependencies=yes``` will automatically be given ```After=basic.target```.

You can see for yourself with ```systemctl --user show dbus | grep ^After```.

I realized it's also in the manual: ```man systemd.special```.